apps/antalmanac/src/lib/utils.ts

```diff
@@ -0,0 +1,30 @@
+export function notNull<T>(value: T): value is NonNullable<T> {
```

Comment:
how is this not built in to the language 😭 also, `!=` instead of `!==` is on purpose here?

Reply:
Yes, `!=` will match with `null` and `undefined`, which is synonymous for this case. `!==` will match only with null, but not undefined because JavaScript feature :tm: